src/components/GraphFilter/GraphFind.tsx

```diff
@@ -360,10 +360,10 @@ export class GraphFind extends React.PureComponent<GraphFindProps> {
       case 'name': {
         const isNegation = op.startsWith('!');
         if (disjunctive && isNegation) {
-          MessageCenterUtils.add(`Find values do not allow OR expressions with "not find by name": [${expression}]`);
+          MessageCenterUtils.add(`Can not use 'OR' with 'not find by name'`);
```

Comment:
to be honest, I don't understand this message. not the previous version and not the current.